src/api/AccountApi.ts

```diff
@@ -262,4 +263,26 @@ export class AccountApi extends BaseApi {
         return this.createTransfer(credentials, account, amount, memo, encrypted, fee).pipe(flatMap((operation) =>
             this.api.broadcastApi.broadcastWithCallback(credentials.keyPair, [operation])));
     }
+
+    /**
+     * Creates new account.
+     *
+     * @param credentials registrar account credentials
+     * @param registrar registrar account id
+     * @param name name of new account
+     * @param publicKey publicKey for new account
+     * @param fee {@link AssetAmount} fee for the operation, if left undefined the fee will be computed in DCT asset
+     *
+     * @return a transaction confirmation
+     */
+    public createNew(
+        credentials: Credentials,
+        registrar: ChainObject,
```

Comment:
is not registrar already provided as credentials.account ?

Reply:
Fixed